RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc

```diff
@@ -531,9 +531,9 @@ buildSuperCluster(CalibClusterPtr& seed,
       for( auto i_ps = clustops.first; i_ps != clustops.second; ++i_ps) {
 	edm::Ptr<reco::PFCluster> psclus(i_ps->second);
 #ifdef EDM_ML_DEBUG
-	auto found_pscluster = std::find_if(new_sc.preshowerClustersBegin(),
+	auto found_pscluster = std::find(new_sc.preshowerClustersBegin(),
 					 new_sc.preshowerClustersEnd(),
-					 [&i_ps](const auto& i) { return i.key() == i_ps->first;});
+					 i_ps->second);
```

Comment:
please revert back the changes in this block to what's in the baseline IB used for this PR.

This was introduced apparently during attempts to rebase and placed into a large commit 6a773c5b49a39d93e3862e06eda6d11d8936776d. So, no easy way to reset.